tests/unit/multi-currency/test-class-compatibility.php

```diff
@@ -51,11 +51,19 @@ public function test_override_selected_currency_return_true() {
 
 	public function test_should_hide_widgets_return_false() {
 		$this->mock_wcs_cart_contains_renewal( false );
+		$this->mock_wcs_cart_contains_resubscribe( false );
 		$this->assertFalse( $this->compatibility->should_hide_widgets() );
 	}
 
-	public function test_should_hide_widgets_return_true() {
+	public function test_should_hide_widgets_return_true_test_1() {
```

Comment:
What do you think about naming these tests `test_should_hide_widgets_return_true_when_renewal_in_cart` and `test_should_hide_widgets_return_true_when_resubscribe_in_cart`?

Reply:
This is where my lack of experience gets me, I did the same in #2318 (which was before this PR) due to the number of tests that were so similar, and carried through to this PR with the same. Thanks for the suggestions, I've updated them in https://github.com/Automattic/woocommerce-payments/pull/2367/commits/8d55540622e6a1b5f52c9ff1b2d876e2fa137fc3